src/EFCore.PG/Storage/Internal/Mapping/NpgsqlListTypeMapping.cs

```diff
@@ -194,7 +200,7 @@ class SingleDimComparerWithEquals<TElem> : ValueComparer<List<TElem>>
             public SingleDimComparerWithEquals() : base(
                 (a, b) => Compare(a, b),
                 o => o.GetHashCode(), // TODO: Need to get hash code of elements...
-                source => DoSnapshot(source)) {}
+                source => DoSnapshot(source)) { }
```

Comment:
nit: have seen these added spaces before, can you please configure your IDE not to do this? It makes PRs harder to review (and I prefer the space-less version).

Reply:
Ah, yes -- should be fixed going forward.